public/javascripts/Gallery/src/validation/ValidationMenu.js

```diff
@@ -184,10 +184,11 @@ function ValidationMenu(refCard, gsvImage, cardProperties, modal, onExpandedView
     function _validateLabel(action, thumbsClick) {
         // Log how the user validated (thumbs vs on-card menu) and what option they chose.
         let actionStr;
-        if (onExpandedView && thumbsClick) actionStr = 'Validate_ThumbsExpandedMenuClick';
-        else if (onExpandedView && !thumbsClick) actionStr = 'Validate_ExpandedMenuClick';
-        else if (!onExpandedView && thumbsClick) actionStr = 'Validate_ThumbsMenuClick';
-        else if (!onExpandedView && !thumbsClick) actionStr = 'Validate_MenuClick';
+        let sourceStr;
+        if (onExpandedView && thumbsClick) actionStr = 'Validate_ThumbsExpandedMenuClick', sourceStr = "GalleryExpanded";
```

Comment:
We're already here, we may as well differentiate between thumbs and the usual menu buttons! Something like "GalleryExpandedThumbs" and "GalleryExpandedImage" (image because the buttons are overlaid over the image). Definitely let me know if you have naming suggestions bc I'm not super happy with that one, but I think it gets the job done at least. Of course doing the same for "GalleyCard".

Reply:
I think these names will suffice, pretty straight forward for the exact source since there are layers to validating on the gallery page.